test_container/tests/test/python3/all/sys_executable.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+from exasol_python_test_framework import udf
+
+
+class SysExecutableTest(udf.TestCase):
+    def setUp(self):
+        self.schema = "SYS_EXECUTABLE"
+        self.query(f'CREATE SCHEMA {self.schema}', ignore_errors=True)
+        self.query(f'OPEN SCHEMA {self.schema}', ignore_errors=True)
+
+    def tearDown(self):
+        self.query(f'DROP SCHEMA {self.schema} CASCADE', ignore_errors=True)
+
+    def test_getuser(self):
+        self.query(udf.fixindent('''
```

Comment:
Maybe add a comment:
* Exasol UDFs are using a C-Library implementing a Python interpreter
* `sys.executable` is an interface in Python allowing to get the path to the executable allowing to run Python code
*